src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.ShowCommand;
+import duke.command.TodoCommand;
+import duke.exception.DukeException;
+import duke.exception.WrongInputException;
+
```

Comment:
As mentioned, maybe adding some JavaDoc comments to meet our coding standard when you have the time :) ?

Reply:
Hello, for clarification, do you mean JavaDoc for the class?